docs/src/main/tut/freemonad.md

```diff
@@ -424,7 +424,7 @@ The above forgetful functor takes a `Monad` and:
  - finally keeps the *functor* part (e.g. the `map` function)
 
 By reversing all arrows to build the left-adjoint, we deduce that the
-forgetful functor is basically a construction that:
+free functor is basically a construction that:
```

Comment:
I think in this context the author really meant this to read *free monad*. Thanks for pointing out the problem. Does this make sense to you?